contracts/BaseStrategy.sol

```diff
@@ -241,6 +241,15 @@ abstract contract BaseStrategy {
      */
     function exitPosition() internal virtual;
 
+    /*
+     * Vault calls this function after shares are created during `Vault.report()`.
+     * You can customize this function to any share distribution mechanism you want.
+     */
+    function distributeRewards(uint256 _shares) external virtual {
```

Comment:
if the strategist doesnt override this function all shares would be sent to the strategist?

Reply:
Yes, exactly! Current behavior today. Or, override and split to multiple parties. Unwrap some and split to another token. Burn it all! Possibilities are endless!